config/services.yml

```diff
@@ -28,6 +28,15 @@ earthdata:
 status_app_root: "https://status.earthdata.nasa.gov/api/v1/"
 cwic_root: "http://cwic.wgiss.ceos.org/"
 
+#Earth Data Search Client specific configs
```

Comment:
Should be "Earthdata". Also, let's explain a little better what's going on here. The collection concept IDs included below will be limited to 2000 granules when ordering. Would hurt to also make a note where that limit is set.